syntax = "proto3";

message LocationMessage {
  int32 id = 1;
  int32 person_id = 2;
  string longitude = 3;
  string latitude = 4;
  string creation_time = 5;
}

message LocationMessageListResponse {
  repeated LocationMessage locations = 1;
}

message LocationRequest{
  int32 person_id = 1;
  string start_date = 2;
  string end_date = 3;
  int32 meters = 4;
}

service LocationService {
  rpc Get(LocationRequest) returns (LocationMessageListResponse);
}

